Foundation/Dictionary.swift

```diff
@@ -18,7 +18,8 @@ extension Dictionary : _ObjectTypeBridgeable {
         
         var idx = 0
         
-        self.forEach { (keyItem, valueItem) in
+        self.forEach { (arg) in
```

Comment:
Stylistic point: this and all subsequent parameter lists in closures don't need the parentheses around `arg`.